scripts/us_eia/opendata/process/common.py

```diff
@@ -82,11 +234,13 @@ def _parse_date(d):
         m_or_q = d[4:]
 
         if m_or_q.startswith('Q'):
+            #print("withQ",yr + '-' + _QUARTER_MAP[m_or_q])
             # Quarterly
             if m_or_q in _QUARTER_MAP:
                 return yr + '-' + _QUARTER_MAP[m_or_q]
         else:
             # Monthly
+            #print("withOutQ",yr + '-' + m_or_q)
```

Comment:
Please remove the commented line if it's not required.

Reply:
Removed